tests/testsuite/main.rs

```diff
@@ -132,6 +132,8 @@ mod weak_dep_features;
 mod workspaces;
 mod yank;
 
+find_tests!("tests/snapshots/");
```

Comment:
- The macro is doing more than what the name says.  It isn't just finding tests but importing them

Reply:
Should it only be doing one or do we just want to rename it?

Reply:
Since it can't return results for something else to act on, I'm assuming we should rename it.  Maybe `import_tests`